src/utils.ts

```diff
@@ -71,3 +73,118 @@ export const getCount = (item: number | string) => {
   const formattedItem = typeof item === "string" ? item.length : item;
   return wordsFormat.format(formattedItem);
 };
+
+export class Metadata {
+  private metaData: string;
+  public username: string;
+  public fileTitle: string;
+  public source: string;
+
+  constructor({
+    fileTitle,
+    username,
+    url,
+    date,
+    tags,
+    speakers,
+    categories,
+  }: MetadataProps) {
+    this.username = username;
+    this.fileTitle = fileTitle;
+    this.source = url;
+
+    // eslint-disable-next-line prettier/prettier
+    this.metaData =
+      `---\n` + `title: ${fileTitle}\n` + `transcript_by: ${username} \n`;
```

Comment:
so this isn't quite right. We pass in who the transcript is by in the content payload. We should use that. We can create something new here like `reviewed_by`